Images_And_Graphics_Index1.html

```diff
@@ -134,17 +57,17 @@ <h1>Rangituhi Walk</h1>
   <!-- Full-width images with number and caption text -->
   <div class="mySlides fade">
     <div class="numbertext">1 / 3</div>
-    <img src="Images/wainuiomata-Cp.jpg">
+    <img src="Images/wainuiomata-Cp.jpg" alt="wainuiomata Camp">

```

Comment:
In my first version I forgot to add alt so in my next version I added alt="wainuiomata Camp"